From e5642e46903ea14c0c73fd3a02f93fd29d4fed83 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Thu, 27 Apr 2023 14:35:13 +0200
Subject: [PATCH 0282/1141] arm64: dts: st: add lvds pixel clock to ltdc

Add management of lvds pixel clock to display controller.
This rework will be clean up when the clock framework
will expose a kind of LVDS pixel "virtual" clock.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: I6684c6210b79ead22e8d55cf85d0fb903bd0377f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/305996
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp253.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp253.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp253.dtsi
@@ -115,3 +115,8 @@
 		};
 	};
 };
+
+&ltdc {
+	clocks = <&rcc CK_BUS_LTDC>, <&rcc CK_KER_LTDC>, <&lvds 0>;
+	clock-names = "bus", "lcd", "lvds";
+};
